apps/nav-veileders-flate/src/components/tiltak/EndreDeltakelseKnapp.tsx

```diff
@@ -61,12 +57,32 @@ export const EndreDeltakelseKnapp = () => {
     pamelding.status.type === DeltakerStatusType.FULLFORT ||
     pamelding.status.type === DeltakerStatusType.AVBRUTT
 
+  const deltakerHarAvsluttendeStatus =
```

Comment:
Ville kanskje deklarert noen av disse funksjonene på utsiden av `export const EndreDeltakelseKnapp = () => {` og heller sendt inn parametere. Hver gang man gjør en endring i en komponent vil alle kjøre for å oppdatere status, og disse funksjonene vil da bli redeklarert, noe som kan ha en innvirkning på minnebruk etc.